infrastructure/api/api_test.go

```diff
@@ -0,0 +1,85 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/OrlandoRomo/academy-go-q32021/domain/model"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	testApiKey = "cc2464e4a1mshb5ceeca91e5a6adp1fa80bjsn4b48e2408b87"
+)
+
+func TestUpdateCSVPath(t *testing.T) {
+	t.Run("set valid CSV path", func(t *testing.T) {
+		urbanClient := NewUrbanDictionary(testApiKey)
+		urbanClient.UpdateCSVPath("/usr/local/definitions.csv")
```

Comment:
I don't think you should use absolute paths, what if someone else downloads the repo and tries to run the tests? This could work in your machine but not in others, the csv should be on the repo